### ISSUE TITLE
EFI: Global Process

### ISSUE THREAD

**Issue by sheplu** [expressjs/discussions#180]
### Motivation

The TC is now reformed and we need to continue to recreate the previous community while improving it. This mean creating dedicated roles with specific tasks

### Expectation

Create roles and groups covering the needs of the project

### Implementation

Create or update the Triager group
Create the Captains role
Define new Working Groups
See security process for the project (OpenSSF?)

### Status

Part: Organization

### Draft

> Define specific process for all the teams and groups of people working part of the organization
TC
Triager
Captains
Working Groups
Security

**Comment by wesleytodd** [comment]
There are a few tracks of work here:

1. Triage team: we need to probably have a kick off meeting, discuss if the current description is meeting our needs (I think we might want to change the signup process)
2. Captains: we need to onboard the first round of new ones.
3. Working groups: this overlaps with security. There is https://github.com/expressjs/discussions/issues/165 where we should track that. I think after that we can use the model to see if we need anything more formal (but I am thinking maybe we dont)
4. Security: the audit. Ideally this would be the first thing the WG does

**Comment by sheplu** [comment]
@wesleytodd yes, I am just creating all the issue before being updating everything and adding related topic and more information :)

**Comment by bjohansebas** [comment]
Is there something to do here? We already have several teams, captains have been defined for each package within the organization, and collaborators have been added to each team. We also have a security group and a triage group.

I’m going to close this, but if you think something is missing, feel free to reopen it.
